docs/public/operator-hub.adoc

```diff
@@ -1,271 +1,151 @@
 
-On the other hand, `EtcdCluster` 's definition looks like below:
+=== Deploying etc to a cluster via YAML
 
-[source,yaml]
-----
-apiVersion: etcd.database.coreos.com/v1beta2
-kind: EtcdCluster
-metadata:
-  name: example
-spec:
-  size: 3
-  version: 3.2.13
-----
 
-There's no placeholder data here and it can thus be used to spin a working
-service from the Operator.
+In this example, we will be deploying our link:https://operatorhub.io/operator/etcd[installed etcd Operator] to an OpenShift / Kubernetes cluster.
 
-=== [[create-service-from-yaml]]Create service from a YAML file
+However, we will be using the YAML definition where we modify the `metadata.name` and `spec.size`.
 
-[NOTE]
+[IMPORTANT]
 ====
-This feature is provided on temporary basis while we work on adding support for
-link:https://github.com/openshift/odo/issues/2785[passing parameters on the
-command line] and link:https://github.com/openshift/odo/issues/2799[using
-interactive mode] to create Operator backed services.
+Deploying via YAML is a **temporary** feature as we add support for link:https://github.com/openshift/odo/issues/2785[passing parameters on the command line] and link:https://github.com/openshift/odo/issues/2799[interactive mode].
 ====
 
-If the YAML definition of the service (or Custom Resource) that you want to
-start has placeholder data in its Operator's `metadata`, you can use
-<<dry-run,`--dry-run` feature>> explained above to get the YAML definition,
-replace the placeholder values with correct values and start the service
-using the corrected YAML definition.
-
-For example, if you would like start an `EtcdCluster` service but of a smaller
-size than what's configured by default, you could first fetch the YAML
-definition of the service:
-
+. Retrieve the YAML output of the operator:
++
 [source,shell]
 ----
-$ odo service create etcdoperator.v0.9.4 --crd EtcdCluster --dry-run
+  $ odo service create etcdoperator.v0.9.4 --crd EtcdCluster --dry-run > etcd.yaml
 ----
 
-and then modify the YAML to below:
-
+. Modify the YAML file by redefining the name and size:
++
 [source,yaml]
-.etcd.yaml
 ----
 apiVersion: etcd.database.coreos.com/v1beta2
 kind: EtcdCluster
 metadata:
-  name: my-etcd-cluster // <1>
+  name: my-etcd-cluster // Change the name
 spec: 
-  size: 1 // <2>
+  size: 1 // Reduce the size from 3 to 1
   version: 3.2.13
 ----
-<1> We changed the name from `example` to `my-etcd-cluster`
-<2> We reduced the size from `3` to `1`
-
-Now we can use the `etcd.yaml` file above to create a service:
 
+. Create the service from the YAML file:
++
 [source,shell]
 ----
-$ odo service create --from-file etcd.yaml
+  $ odo service create --from-file etcd.yaml
 ----
 
-This will result in a `EtcdCluster` service with only one pod instead of the
-three pods that it's originally configured to create. This can be checked by
-doing:
-
+. Confirm that the pod has been created:
++
 [source,shell]
 ----
-$ kubectl get pods | grep my-etcd-cluster
+  # OpenShift
```

Comment:
In case you decide to include `odo service list`, below two be replaced with `odo service list`.